examples/model_serving.py

```diff
@@ -0,0 +1,145 @@
+"""
+This example demonstrates using torch.jit.ModuleScript for model deployment.
```

Comment:
ModuleScript -> ScriptModule and add a link https://pytorch.org/docs/stable/jit.html#torch.jit.ScriptModule